File: multify/lib/multify/project.rb
class Multify::Project < Multify::Resource

  extend Multify::Resource::WithSlug

  assignable_attributes :name, :description, :slug

  attribute :name,        String
  attribute :description, String
  attribute :slug,        String
  attribute :created_at,  Time
  attribute :updated_at,  Time

  def tasks
    Tasks.new(id) if persisted?
  end

end

require 'multify/project/tasks'

File: www_static/javascripts/multify/project.js
Multify.Project = Backbone.Model.extend({
  name: null,
  slug: null,

  initialize: function() {
    this.tasks = new Multify.Tasks(this);
  }
});

Multify.Project.modelName = 'project';
Multify.Project.path = 'projects';
